renovate.json

```diff
@@ -243,7 +243,7 @@
         "^task/.+/patch\\.yaml$"
       ],
       "matchStrings": [
-        "(?:image: |tooling-image=)(?:['\"])?(?<depName>[0-9a-z.\/-]+)(?::(?<currentValue>[0-9a-z.-]+))?@(?<currentDigest>sha256:[a-f0-9]{64})(?:['\"])?"
+        "(?:image: |tooling-image=)(?:['\"])?(?<depName>[0-9a-z./-]+)(?::(?<currentValue>[0-9a-z.-]+))?@(?<currentDigest>sha256:[a-f0-9]{64})(?:['\"])?"
```

Comment:
Is this change related to this PR?

Reply:
@tnevrlka you mentioned that dropping the backslash somehow makes the regex invalid? https://github.com/konflux-ci/build-definitions/pull/2154#issuecomment-2769290196

Reply:
@chmeliik I can remove the renovate update for this PR but I guess that it will raise up again when someone will use the `generate-everything.sh` script

Reply:
Ok, let's keep the renovate.json update